content/admin/configuration/configuring-github-connect/about-github-connect.md

```diff
@@ -27,9 +27,9 @@ After enabling {% data variables.product.prodname_github_connect %}, you will be
 
 After you configure the connection between {% data variables.location.product_location %} and {% data variables.product.prodname_ghe_cloud %}, you can enable individual features of {% data variables.product.prodname_github_connect %} for your enterprise.
 
-Feature | Description | More information |
-------- | ----------- | ---------------- |{% ifversion ghes %}
-Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}{% ifversion ghes or ghae %}
+|Feature | Description | More information |
+| ----------- | ----------- | ----------- |
+{% ifversion ghes %} Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}{% ifversion ghes or ghae %}
```

Comment:
Moving this Liquid versioning to the next line as it's breaking the formatting for GHAE.
```suggestion
{% ifversion ghes %} Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}
```

Reply:
See this screenshot: 
![ghae-version](https://user-images.githubusercontent.com/42146119/214315803-16cd4a69-a159-49eb-b4ad-1d74ae6cbcac.png)